### DISCUSSION TITLE
插件问题求助

### DISCUSSION THREAD

**Discussion by TsukasaYahane** [nonebot/discussions#456] [问答] [answered]
跟着教程写的天气插件，只改动了输入地名查询的函数（接入三方API实现实时查询）后，无法通过指令响应插件，断点测试时发送“天气”未经过插件内断点。查询函数本地测试完全可用可排除问题，求大佬们指点一下
![20240421012642 (1)](https://github.com/nonebot/discussions/assets/121178437/e3bd6575-a3dd-4f50-912f-152845fd34c5)
![20240421012642 (2)](https://github.com/nonebot/discussions/assets/121178437/2f003f70-6db5-46e6-8255-92fdbd3097c7)

**Comment by yanyongyu** [comment, marked as answer]
如果你没有调整过command start配置项的话默认是`/天气`

**Reply by TsukasaYahane** [reply]
command_start里的启动命令也试过了"/"以及" "开头均无响应

**Reply by yanyongyu** [reply]
那你应该确认一下插件是否被正确加载，启动日志中可以看见

**Reply by TsukasaYahane** [reply]
重构插件载入模块可正常使用了 感谢大佬指点
